recipes/merqury/meta.yaml

```diff
@@ -1,17 +1,18 @@
 {% set name = "merqury" %}
 {% set version = "1.3" %}
-{% set sha256 = "b6e3f54a6fe30acfe6f2b15ebc26a103a51980adfc03522e712a2baa5e79131a" %}
+{% set sha256 = "42aceca4078b60d3b6f252c6913983745cfe51c0e5be67bfdc6bc145d3572523" %}
+
 
 package:
   name: "{{ name }}"
   version: "{{ version }}"
 
 source:
-  url: https://github.com/marbl/{{name}}/archive/refs/tags/v{{version}}.tar.gz
+  url: https://github.com/marbl/{{name}}/archive/593c9eff18a7dd778cd66f143736f1543b7b5826.tar.gz
```

Comment:
This is not "version 1.3" anymore.

Reply:
How, would you like to call this version?

Reply:
That seems to have been intentional